Coding Guidelines.md

```diff
@@ -0,0 +1,6 @@
+Project Reunion implementations prefer C++ and C++/WinRT implementations.   New code must adhere to the [cppcoreguidelines](https://github.com/isocpp/CppCoreGuidelines) 
```

Comment:
Please steal content from https://github.com/microsoft/ProjectReunion/compare/user/drustheaxe/styleguide 

We want all that content, but better in a coding guidelines.md than in contributor.md. I'll scrap my branch after you assimiliate its meat :-)

Reply:
incorporated the items as discussed offline.  thank you. :-)